.travis.yml

```diff
@@ -17,7 +18,24 @@ before_script:
   - composer selfupdate --quiet
   - composer install --prefer-dist --no-interaction --no-progress
   - ./vendor/bin/phpunit
-  - composer update --prefer-dist --no-interaction --no-progress
 
 script:
-  - ./vendor/bin/phpunit
+  - make test
+
+after_script:
```

Comment:
No : deployment is triggered only on `master` branch. I think that if I use `before_deploy`, tests will not be launch for Pull requests.